distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1748,7 +1760,10 @@ type Table
              an `Unquoted_Delimiter`
            - If there are more than 10 issues with a single column,
              an `Additional_Warnings`.
-    cross_tab : Aggregate_Column | Text | Integer | Column_Selector | Vector (Integer | Text | Column_Selector | Aggregate_Column) -> (Text | Integer) -> Vector Aggregate_Column -> Problem_Behavior -> Table ! Missing_Input_Columns | Column_Indexes_Out_Of_Range | Invalid_Aggregate_Column | Floating_Point_Equality | Invalid_Aggregation | Unquoted_Delimiter | Additional_Warnings
+    @group_by Widget_Helpers.make_column_name_vector_selector
+    @name_column Widget_Helpers.make_column_name_selector
+    @values (Widget_Helpers.make_aggregate_column_selector include_group_by=False)
```

Comment:
Don't we want to use this on `aggregate` too??

Reply:
eventually - this is the first test